_data/workshops.yml

```diff
@@ -1,4 +1,21 @@
 workshops:
+-
+    who: TBA
+    help: TBA
```

Comment:
https://lirgroup.heanet.ie/index.php/2019/03/04/liber-lc-instructortraining2019/ has a few names. If they shall not be added here this PR is good to merge, IMHO

Reply:
Good catch @katrinleinweber! @davidfkane can you add the names of the instructors and helpers? Also, do you have the workshop site up and running so we can link to it?